### PR TITLE
Wiki Startup

### PR DESCRIPTION
**Pull request by v-h-gn**
Additions
------------
- Adds `staff.rst`
- Adds a reST reference on every `.rst` file
- Adds `core` folder
- Adds `core/abilities` folder
- Adds link to repo in `source/index.rst` for issue reporting
- Adds download section to `general/index.rst`
- Adds FAQ section to `general/index.rst`
- Adds contributing section to `general/index.rst`
- Adds captions to the `toctree`
- Adds PK logo to wiki

Removals
------------
- Removes `admin`, `players`, `developers` folders from wiki directory

Misc Changes
----------------
- Changes `general/index.rst` to include general info about PK
- Separates wiki specific contribution to own specific page (`wikicontribute.rst`)
- Moves `commands.rst` from `general` folder to `core` folder
- Renames and moves `pkcore.rst` to `core/index.rst`
- Fixes various grammar errors.
- Changes various descriptions for versatility and style purposes
- Changes `toctree: maxdepth:` to 1.

### CODE REVIEW

**Comment by v-h-gn** [comment]
Images will be shortly added to the staff.rst, I just need people to respond to the form I sent in general.

File: source/core/development/customabil.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+.. _customabil:
```
Comment (Simplicitee) [inline source/core/development/customabil.rst]: Useless

File: source/core/index.rst [hunk 2]
```diff
@@ -37,12 +41,19 @@ it's up to the player to find the one that suits them the most.
    placed kicks and punches to block chi paths in bender's bodies, doing
    anything from preventing them from bending to temporarily paralyzing
    them.
+   
+:ref:`Commands and Permissions <commands>`
```
Comment (Simplicitee) [inline source/core/index.rst:47]: Referencing an explicit link puts the title, don't need to write it out yourself

File: source/general/index.rst [hunk 3]
```diff
@@ -1,80 +1,87 @@
...
+  * `Core`_
 
-    pip install -r .\requirements.txt
+* Modded
 
-This should install both sphinx as well as the theme.
+  * `ATLA Mod`_
 
-To check that sphinx is correctly install run:
+**\*PROJECTKORRA IS NOT RESPONSIBLE FOR ERRORS CAUSED BY NON PROJECTKORRA RESOURCES RESOURCES DOWNLOADED FROM OUTSIDE OF THE PROJECTKORRA FORUMS ARE ALSO NOT INSPECTED BY PROJECTKORRA AND MAY CONTAIN MALICIOUS CODE**
```
Comment (Simplicitee) [inline source/general/index.rst]: a. reread this
b. it'd look better as a note

File: source/general/index.rst [hunk 4]
```diff
@@ -1,80 +1,87 @@
...
+* Vanilla
 
-.. code:: bash
+  * `Core`_
 
-    pip install -r .\requirements.txt
+* Modded
 
-This should install both sphinx as well as the theme.
+  * `ATLA Mod`_
```
Comment (Simplicitee) [inline source/general/index.rst]: this downloads section has a lot of problems
a. Spigot is technically a Mod so it's not Vanilla
b. The name of the mod stuff is Forge
c. ATLA Mod, it has a name. Avatar Mod 2: Out of the Iceberg

File: source/general/index.rst [hunk 5]
```diff
@@ -1,80 +1,87 @@
...
 
 Useful Links
-============
\ No newline at end of file
+==============
+
+- `Website`_
+- `Discord`_
+- `Twitter`_
+- `GitHub`_
```
Comment (Simplicitee) [inline source/general/index.rst]: These links are boring and not very informational. View mine to understand what I mean better.

File: source/general/staff.rst [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+
+Junior Developers
+^^^^^^^^^^^^^^^^^^^
+|varhagna| Varhagna
+
+Mod Developers
+^^^^^^^^^^^^^^^^
+|favouritedragon| FavouriteDragon
+|mahtaran| Mahtaran
+|mnesikos| Mnesikos
```
Comment (Simplicitee) [inline source/general/staff.rst]: who?

File: source/general/staff.rst [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+.. |mnesikos| image:: mnesi.png
+.. |coolade| image:: coolade.png
+.. |phanaticd| image:: phanatic.png
+.. |pickle9775| image:: pickle.png
+.. |plasmarob| image:: plasma.png
+.. |mist| image:: mist.png
+.. |simp| image:: simp.png
+.. |strangeone101| image:: strange.png
+.. |thewaterchief| image:: thechief.png
+.. |xitzniek| image:: itzniek.png
```
Comment (Simplicitee) [inline source/general/staff.rst]: not used

File: source/index.rst [hunk 8]
```diff
@@ -5,49 +5,27 @@
...
-.. toctree::
-    :maxdepth: 2
-
-    developers/index
+    :maxdepth: 1
+    :caption: Core
+    :name: core
+    
+    Home <core/index>
+    Commands and Permissions <core/commands>
```
Comment (Simplicitee) [inline source/index.rst:30]: Renaming them huh? Kinda complained about me doing that...

File: source/index.rst [hunk 9]
```diff
@@ -5,49 +5,27 @@
...
---------------
-
-This section help developers understand ProjectKorra and contain development tutorials.
-
-.. toctree::
-    :maxdepth: 2
-
-    developers/index
+    :maxdepth: 1
+    :caption: Core
```
Comment (Simplicitee) [inline source/index.rst:26]: ProjectKorra Core / Core Plugin would be a better caption

**Review by Simplicitee — COMMENTED** [review]
Also missing the abilities section Chris specifically asked for

File: source/core/development/customabil.rst (hunk 1 above)
Comment (vtovmasian) [reply source/core/development/customabil.rst]: Agreed, I was planning on removing it

Comment (vtovmasian) [reply source/core/development/customabil.rst]: Agreed, I was planning on removing it

File: source/core/index.rst (hunk 2 above)
Comment (vtovmasian) [reply source/core/index.rst:47]: Yes it does, however I wrote out all of my references incase I felt that it should be changed

File: source/general/index.rst (hunk 4 above)
Comment (vtovmasian) [reply source/general/index.rst]: 1. Okay, Ill change it to Spigot
2. I know it has a name, but its lengthy and I wanted to shorten it, Ill figure out something better and change it.

File: source/general/index.rst (hunk 5 above)
Comment (vtovmasian) [reply source/general/index.rst]: Its pretty self explanatory, but fine

File: source/general/staff.rst (hunk 7 above)
Comment (vtovmasian) [reply source/general/staff.rst]: It will be used. Im still working on it

File: source/general/staff.rst (hunk 6 above)
Comment (vtovmasian) [reply source/general/staff.rst]: See Slack

File: source/index.rst (hunk 8 above)
Comment (vtovmasian) [reply source/index.rst:30]: That was not my specific issue. I'll explain further in which situation I think renaming is bad

File: source/general/staff.rst (hunk 7 above)
Comment (Simplicitee) [reply source/general/staff.rst]: I only meant the niek guy

File: source/index.rst (hunk 9 above)
Comment (vtovmasian) [reply source/index.rst:26]: I prefer the names to be short.
The first version looks better imo
CORE --> PROJECTKORRA CORE
RPG --> PROJECTKORRA RPG
ITEMS --> PROJECTKORRA ITEMS

File: source/general/staff.rst (hunk 7 above)
Comment (vtovmasian) [reply source/general/staff.rst]: See Slack

File: source/general/index.rst (hunk 3 above)
Comment (vtovmasian) [reply source/general/index.rst]: I don't like how the notes look, but fine.

**Pull request closed without merging**